src/intern/intern-saucelabs.js

```diff
@@ -0,0 +1,30 @@
+define([
+	'./intern'
+], function (intern) {
+
+    intern.capabilities = {
+        project: '<%- appName %>',
+        name: '<%- appName %>',
+        fixSessionCapabilities: false
+    };
+
+    intern.environments = [
+        { browserName: 'internet explorer', version: [ '10.0', '11.0' ], platform: 'Windows 7' },
+        // { browserName: 'microsoftedge', platform: 'Windows 10' },
```

Comment:
In some cases yes. For a fresh out of the box sample app it works and adding one or two dummy test cases didn't seem to break it. But in stores or core it still has issues.